### PR TITLE
Partial admin commit

### PR DESCRIPTION
**Pull request by ngobert**
Updated commit module to include parameter for partial admin commit.  Dependent upon pandevice changes in pandevice PR 173 (https://github.com/PaloAltoNetworks/pandevice/pull/173).

Addresses feature request #401  .

### CODE REVIEW

File: module_utils/network/panos/panos.py [hunk 1]
```diff
@@ -433,12 +433,17 @@ def commit(self, module, include_template=False):
...
             include_template (bool): (Panorama only) Force include the template.
+            admins (list): This is the list of admins whose changes will be committed to
+                the firewall/Panorama. The admins argument works with PanOS 8.0+. 
         """
         if module.check_mode:
             return
 
         try:
-            self.device.commit(sync=True, exception=True)
+            if admins is not None:
```
Comment (shinmog) [inline module_utils/network/panos/panos.py]: Don't need an if/else here; just add in `, admins=admins` to the function call.  It'll already be `None` if the user didn't specify anything.

File: examples/pano_commit.yml [hunk 2]
```diff
@@ -17,3 +27,9 @@
       username: '{{ username }}'
       password: '{{ password }}'
       devicegroup: 'shared_services_11022'
+
+  - name: Commit pending Panorama changes for specified admin list
+    panos_commit:
+      provider: "{{ panos_provider }}"
+      admins: "{{ admins }}"
+
```
Comment (shinmog) [inline examples/pano_commit.yml]: Remove this extra newline, so there's only a single newline at the end of the file.

File: module_utils/network/panos/panos.py (hunk 1 above)
Comment (ngobert) [reply module_utils/network/panos/panos.py]: @shinmog - The last push has addressed both of these comments.  Please let me know if you have any other feedback or comments.  Thanks!

**Comment by ngobert** [comment]
Thank you very much for the thorough review and assistance.  I still owe
you that URL category module...  I got sidetracked on some other projects
so didn't get around to rebuilding it after you guys switched to the helper
approach.

Nick

**Pull request merged**